CoffeeMachineSimulator/CoffeeMachineSimulator.Tests/Services/CoffeeServiceTests.cs

```diff
@@ -33,11 +37,52 @@ public void GetCoffees_Returns_ListOfCoffeeModels()
         public void AddNullCoffee_Throws_NewException()
         {
             var ex = Assert.Throws<Exception>(() => coffeeService.AddCoffee(null));
-            
+
             Assert.AreEqual(ex.Message, "You should not add null entries!");
         }
 
         //TODO: Add Unit test for Remaining add cases
+        [Test]
+        public void AddCoffeWithoutPrice_WillNotAddCoffe()
+        {
+            var initial_Length_List = coffeeService.GetCoffees().Count;
```

Comment:
do not use special characters in variables naming. Also, a better naming would be : initialCoffeesLenght